pylxd/tests/models/test_container.py

```diff
@@ -431,13 +431,39 @@ def setUp(self):
         super(TestFiles, self).setUp()
         self.container = models.Container.get(self.client, 'an-container')
 
-    def test_put(self):
-        """A file is put on the container."""
+    def test_put_delete(self):
+        """A file is put on the container and then deleted"""
         data = 'The quick brown fox'
 
-        self.container.files.put('/tmp/putted', data)
-
-        # TODO: Add an assertion here
+        res = self.container.files.put('/tmp/putted', data)
+        self.assertEqual(True, res, msg='Failed to put file, result: {}'.format(res)) # NOQA
+
+        # if we are mocked, delete should initially not be available
+        if self.is_mocked():
```

Comment:
Shouldn't need this `if self.is_mocked()` test; the service will be mocked as the test case is derived from `...testing.PyLXDTestCase`.